index.html

```diff
@@ -263,7 +264,13 @@ <h3>
             <li>Presentation Request Data Model</li>
             <li>Storage and Sharing of Verifiable Credentials</li>
             <li>Privacy Guidance for Verifiable Credentials</li>
-            <li>An API for Verifiable Credential Exchange</li>
+            <li>A Developer Guide consisting of one or more notes related to:
+              <ul>
+                <li>General implementaion guidance and best practices for working with VCs</li>
+                <li>One or more HTTP protocol definitions for Verifiable Credential Exchange (such as the <a href="https://w3c-ccg.github.io/vc-api/">VC-API</a>)</li>
```

Comment:
```suggestion
                <li>Discussion of one or more HTTP protocol definitions for Verifiable Credential Exchange (such as the <a href="https://w3c-ccg.github.io/vc-api/">VC-API</a>)</li>
```

Reply:
I don't really follow this suggested change

The original wording implies that we will work on creating a developer guide around one or more HTTP protocols, the suggested change implies we will work on creating a developer guide around discussing these protocols.

Reply:
I also think the original wording was more clear. I might also suggest just "HTTP protocol definition(s)..." because the sentence leading to this list already has "one or more" and "notes about" ... Meaning there will be a Developer Guide with one or more notes on related HTTP protocols...